### PR TITLE
PR for middle node in LinkedList

### PR DESCRIPTION
**Pull request by shahabrar7746**
- [*] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [*] This pull request is all my own work -- I have not plagiarized it.
- [*] All filenames are in PascalCase.
- [*] All functions and variable names follow Java naming conventions.
- [*] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

File: src/main/java/com/thealgorithms/datastructures/lists/SinglyLinkedList.java [hunk 1]
```diff
@@ -16,6 +16,19 @@ public class SinglyLinkedList extends Node {
      * Size of SinglyLinkedList
      */
     private int size;
+    
+    //below is the function to find out middle node in the singlyLinkedList.
+      public Node middle(){
+    
+  Node s = head;
+    Node f = s.next;
+    while(f != null && f.next != null){
```
Comment (quangdutran) [inline src/main/java/com/thealgorithms/datastructures/lists/SinglyLinkedList.java:25]: If you could make these lines aligned with the existing format, that would be awesome

**Comment by quangdutran** [comment]
> The code looks good, could you please add some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

I could help with the tests, is it possible to contribute commit to this PR?

**Comment by siriak** [comment]
I don't think so, you'd need to make a fork and make changes there

File: src/main/java/com/thealgorithms/datastructures/lists/SinglyLinkedList.java (hunk 1 above)
Comment (shahabrar7746) [reply src/main/java/com/thealgorithms/datastructures/lists/SinglyLinkedList.java:25]: can you tell me, how can i add Juits to this?

**Pull request closed without merging**